### PR TITLE
[Docs] maintainers_guidelines: add WG and real-time chat request info

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
[Docs] maintainers_guidelines: add WG and real-time chat request info

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
docs/docsite/rst/community/maintainers_guidelines.rst

### CODE REVIEW

File: docs/docsite/rst/community/maintainers_guidelines.rst [hunk 1]
```diff
@@ -52,6 +52,21 @@ Collection contributors and maintainers should also communicate through:
 
 See :ref:`communication` for more details on these communication channels.
 
+.. _wg_and_real_time_chat:
+
+Working Group and Real-time Chat Request
+----------------------------------------
+
+To make communication within your community more efficient and interactive, you can:
```
Comment (oraNod) [inline docs/docsite/rst/community/maintainers_guidelines.rst]:
```suggestion
Here are some techniques for communicating more efficiently and interactively within your community:
```

File: docs/docsite/rst/community/maintainers_guidelines.rst [hunk 2]
```diff
@@ -52,6 +52,21 @@ Collection contributors and maintainers should also communicate through:
...
+
+* Join existing suitable :ref:`working_group_list` and corresponding real-time chats if exist.
```
Comment (oraNod) [inline docs/docsite/rst/community/maintainers_guidelines.rst]:
```suggestion
* Find an existing :ref:`working_group_list` that is similar to your project and join the conversation.
```

File: docs/docsite/rst/community/maintainers_guidelines.rst [hunk 3]
```diff
@@ -52,6 +52,21 @@ Collection contributors and maintainers should also communicate through:
...
+* `Request <https://github.com/ansible/community/blob/main/WORKING-GROUPS.md>`_ a new working group for your project.
+* `Create <https://hackmd.io/@ansible-community/community-matrix-faq#How-do-I-create-a-public-community-room>`_ a real-time chat for your working group or `ask <https://github.com/ansible/community/issues/new>`_ the community team to create it and make it official.
```
Comment (oraNod) [inline docs/docsite/rst/community/maintainers_guidelines.rst]:
```suggestion
* `Create <https://hackmd.io/@ansible-community/community-matrix-faq#How-do-I-create-a-public-community-room>`_ a public chat for your working group or `ask <https://github.com/ansible/community/issues/new>`_ the community team.
```

File: docs/docsite/rst/community/maintainers_guidelines.rst [hunk 4]
```diff
@@ -52,6 +52,21 @@ Collection contributors and maintainers should also communicate through:
...
+* Add information about the working group and related real-time chats to your project's ``README.md`` and to its contributor documentation.
```
Comment (oraNod) [inline docs/docsite/rst/community/maintainers_guidelines.rst]:
```suggestion
* Provide working group details and links to chat rooms in the contributor section of your project ``README.md``.
```

File: docs/docsite/rst/community/maintainers_guidelines.rst [hunk 5]
```diff
@@ -64,13 +79,12 @@ Share your opinion and vote on the topics to help the community make the best de
...
 
 Weekly community Matrix/IRC meetings
 ------------------------------------
 
-The Community and the Steering Committee come together at weekly meetings in the ``#ansible-community`` :ref:`Matrix/Libera.Chat <communication_irc>` channel to discuss important project-scale questions. See the `schedule <https://github.com/ansible/community/blob/main/meetings/README.md#schedule>`_ and join.
+The Community and the Steering Committee come together at weekly meetings in the ``#ansible-community`` `Libera.Chat IRC <https://docs.ansible.com/ansible/devel/community/communication.html#ansible-community-on-irc>`_ channel or in the bridged `#community:ansible.com <https://matrix.to/#/#community:ansible.com>`_ room on `Matrix <https://docs.ansible.com/ansible/devel/community/communication.html#ansible-community-on-matrix>`_ to discuss important project-scale questions. See the `schedule <https://github.com/ansible/community/blob/main/meetings/README.md#schedule>`_ and join.
```
Comment (oraNod) [inline docs/docsite/rst/community/maintainers_guidelines.rst]:
```suggestion
The Community and the Steering Committee come together at weekly meetings in the ``#ansible-community`` `Libera.Chat IRC <https://docs.ansible.com/ansible/devel/community/communication.html#ansible-community-on-irc>`_ channel or in the bridged `#community:ansible.com <https://matrix.to/#/#community:ansible.com>`_ room on `Matrix <https://docs.ansible.com/ansible/devel/community/communication.html#ansible-community-on-matrix>`_ to discuss important project questions. Join us! Here is our `schedule <https://github.com/ansible/community/blob/main/meetings/README.md#schedule>`_.
```

File: docs/docsite/rst/community/maintainers_guidelines.rst [hunk 6]
```diff
@@ -82,6 +96,7 @@ Expanding the collection community
 Here are some ways you can expand the community around your collection:
 
   * Give :ref:`newcomers a positive first experience <collection_new_contributors>`.
+  * Offer them to join related :ref:`real-time chats <wg_and_real_time_chat>`.
```
Comment (oraNod) [inline docs/docsite/rst/community/maintainers_guidelines.rst]:
```suggestion
  * Invite contributors to join :ref:`real-time chats <wg_and_real_time_chat>` related to your project.
```

**Comment by Andersson007** [comment]
@oraNod thanks for the suggestion, now it's much better! PTAL

File: docs/docsite/rst/community/maintainers_guidelines.rst [hunk 7]
```diff
@@ -52,6 +52,21 @@ Collection contributors and maintainers should also communicate through:
 
 See :ref:`communication` for more details on these communication channels.
 
+.. _wg_and_real_time_chat:
+
+Working Group and Real-time Chat Request
+----------------------------------------
+
+Here are some techniques for communicating more efficiently and interactively within your community:
```
Comment (oraNod) [inline docs/docsite/rst/community/maintainers_guidelines.rst]:
```suggestion
Establishing working group communication
----------------------------------------------------------------

Working groups depend on efficient, real-time communication.
Project maintainers can use the following techniques to establish communication for working groups:
```

Comment (oraNod) [reply docs/docsite/rst/community/maintainers_guidelines.rst]: How about this? The heading as-is doesn't seem to set the context quite right imo.

Comment (Andersson007) [reply docs/docsite/rst/community/maintainers_guidelines.rst]: i wasn't sure if chats is the only way they can communicate but as there's nothing else in communication guide Working Groups section (e.g. no mailing list, etc, only matrix/irc) I think it's fair to have `chat communication` in the title.
Should we put a blank line between lines 60 and 61 for readability?

Comment (oraNod) [reply docs/docsite/rst/community/maintainers_guidelines.rst]: yeah maybe "chats" isn't quite right. `s/Establishing chat communication/Establishing communication`

I put each sentence on a separate line as a force of habit. ventilated prose ftw. I don't think a blank line is really necessary.

Comment (Andersson007) [reply docs/docsite/rst/community/maintainers_guidelines.rst]: wfm, committed, PTAL

**Review by oraNod — APPROVED** [review]

**Comment by oraNod** [comment]
Thanks @Andersson007

**Comment by Andersson007** [comment]
@oraNod thanks for reviewing!

**Pull request merged**